Javascript/homework/responses/2.3-clock-design.md

```diff
@@ -4,31 +4,34 @@
 
 After you're satisfied with the body of your page, define a selector that styles elements in the `clock` class.
 
-![clock-selector](https://user-images.githubusercontent.com/32557138/106408305-39f9f600-640c-11eb-896d-f46a14259227.png)
+<details><summary>What is a selector?</summary>
+<p>
 
-- Set a width and a height for the box that's going to hold your time  
-- Add a background and a solid border using colors different from the screen's background so you can see what the box will look like.
-  - Use the `background-color` and `border` attributes for these items respectively
+#### CSS selectors are used to "find" (or select) the HTML elements (which is going to be our time div) that you want to style.
 
-- Set your `display` to `flex` 
-- Center your text using `align-items` and `justify-content` 
+```css
+.Clock{
 
-You need to do this again because these attributes in the body only centered the box on your page; repeating the two bullets above in the clock section will center your text within the box.
-
-You may also want to adjust the margins of your clock so that theres some space between the box and the form - I simply adjusted the left margin of my clock.
+}
+```
 
-You can add as many bells and whistles to this clock that you please.  I increased the font-size here and added a box-shadow to add some depth to my clock.
+</p>
+</details>
 
-![clock-color](https://user-images.githubusercontent.com/32557138/106408362-4b430280-640c-11eb-9247-75e983d93da1.png)
+### Once you have a selector that is defineing .clock we're going to move onto the steps below.
```

Comment:
Check for spelling errors / typos before making a PR (copy the markdown file in a google doc or there might be some chrome extensions that can help with this as well)